timeline-chart/src/time-graph-render-controller.ts

```diff
@@ -0,0 +1,58 @@
+import { throttle, debounce } from 'lodash';
+import * as PIXI from 'pixi.js-legacy';
+
+const START_RENDER_STRING = 'startPixiRender';
+const STOP_RENDER_STRING = 'stopPixiRender';
+interface RenderEvents {
+    startRender: () => void;
+    stopRender: () => void;
+}
+
+const throttledStart = throttle(() => {
+    window.dispatchEvent(new Event(START_RENDER_STRING));
+}, 450, { leading: true });
+
+export const RenderEvents: RenderEvents = {
+    /**
+     * Fires an event that will start the PIXI.Ticker.
+     * This will start rendering.
+     * Event handler located in time-graph-render-controller.ts
+     */
+    startRender: () => {
+        throttledStart();
+    },
+    /**
+     * Fires an event that will stop the PIXI.Ticker.
+     * This will stop rendering.
+     * StopRender event handler is debounced @ 1.5s.
+     * Event handler located in time-graph-render-controller.ts
+     */
+    stopRender: () => {
+        window.dispatchEvent(new Event(STOP_RENDER_STRING));
```

Comment:
For consistency, this could be a debouncedStop() method that dispatches the event, and the event listener then would stop the ticker immediately.